src/main/java/tc/oc/pgm/api/discord/DiscordUser.java

```diff
@@ -0,0 +1,15 @@
+package tc.oc.pgm.api.discord;
+
+import jdk.internal.jline.internal.Nullable;
+
+public interface DiscordUser {
```

Comment:
Properly document these methods with javadoc fences (see https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format for a tutorial). Regular comments are not available in javadocs.